New changes

ztalha6 opened the pull request.

rvndsngwn commented:
There are a lot of changes that need to be reviewed. Could you describe the changes you made?

ztalha6 commented:
Hi @rvndsngwn, I made the following changes to the package.

- The UI of the component matters when it comes to usability. I updated the screen UI so that it can be used in most projects.
- Secondly, and most important. I updated the dependencies to make the plugin work in longer run.
